lib/vanagon/platform/windows.rb

```diff
@@ -245,6 +291,7 @@ def initialize(name)
         @install = "/usr/bin/install"
         @copy = "/usr/bin/cp"
         @package_type = "msi"
+        @proj_root = Pathname.new(".").realpath
```

Comment:
This feels really forced to me. If we need the project root directory, that should be a part of the project object, not the platform object